pkg/deploy/strategy/recreate/recreate.go

```diff
@@ -106,13 +106,11 @@ func (s *RecreateDeploymentStrategy) Deploy(deployment *kapi.ReplicationControll
 
 	// Execute any post-hook. Errors are logged and ignored.
 	if params != nil && params.Post != nil {
-		// TODO: handle this in defaulting/conversion/validation?
-		if params.Post.FailurePolicy == deployapi.LifecycleHookFailurePolicyAbort {
-			params.Post.FailurePolicy = deployapi.LifecycleHookFailurePolicyIgnore
-		}
-		err := s.hookExecutor.Execute(params.Post, deployment, s.retryPeriod)
+		err := s.hookExecutor.Execute(params.Post, deployment, "posthook")
 		if err != nil {
-			glog.Infof("Post hook failed: %s", err)
+			glog.Errorf("Post hook failed: %s", err)
```

Comment:
As of now, it just logs - so no different. We can let this be and, if required, this can be picked up by the other issue that addresses this issue (glog.Errorf -> util.HandleError) specifically.